CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
+
+### Bug Fixes
+
+ * "Insert element" code actions don't add close tags when auto close tag is disabled. See [lemminx#1458](https://github.com/eclipse/lemminx/issues/1458).
+ * Improve "Insert only required expected elements" by taking care of choice. See [lemminx#1448](https://github.com/eclipse/lemminx/issues/1448).
+ * Linked editing breaks start tag when closing tag manually. See [lemminx#1456](https://github.com/eclipse/lemminx/issues/1456).
+ * Ignore linked editing range when there are no referenced node to update. See [lemminx#1453](https://github.com/eclipse/lemminx/pull/1453).
+ * Formatting selection fails with root element with mixed content. See [lemminx#1414](https://github.com/eclipse/lemminx/issues/1414).
+ * `SAXParseException` in the language server trace when editing an `.rng` file. See [lemminx#1441](https://github.com/eclipse/lemminx/issues/1441).
+ * Ensure attributes on the first line are wrapped correctly. See [lemminx#1439](https://github.com/eclipse/lemminx/issues/1439).
+ * Fix comment formatting with `xml.format.maxLineWidth`. See [#851](https://github.com/redhat-developer/vscode-xml/issues/851).
+ * Permit binding to a schema from an empty document. See [#819](https://github.com/redhat-developer/vscode-xml/issues/819).
+ * RelaxNG validation with `XInclude` / File association report `DOCTYPE` error. See [#845](https://github.com/redhat-developer/vscode-xml/issues/845).
+ * DocumentLink support for `xi:include/@href`. See [lemminx#1401](https://github.com/eclipse/lemminx/issues/1401).
+ * Binary server doesn't show "element missing child" warning in broken RelaxNG. See [lemminx#1460](https://github.com/eclipse/lemminx/issues/1460).
+ * Refactor "Register catalog" code lens to be in catalog package. See [lemminx#1417](https://github.com/eclipse/lemminx/pull/1417).
+
+### Build
+
+ * Move to `vscode-languageclient` 8.0.1. See [#711](https://github.com/redhat-developer/vscode-xml/issues/711).
+ * Publish a generic vsix for all other platforms. See [#865](https://github.com/redhat-developer/vscode-xml/issues/865).
+ * React to `vsce` renaming to `@vscode/vsce`. See [#857](https://github.com/redhat-developer/vscode-xml/pull/857), [#858](https://github.com/redhat-developer/vscode-xml/pull/858).
+ * Use commit SHA-1 for pinning untrusted GitHub actions to a version. See [#864](https://github.com/redhat-developer/vscode-xml/pull/864).
+
+### Documentation
+
+ * Fix `lint.yaml` build status badge. See [#852](https://github.com/redhat-developer/vscode-xml/pull/852).
+ * Update documentation for generating RelaxNG from XML. See [#848](https://github.com/redhat-developer/vscode-xml/pull/848).
+ * Declare `xml.references` settings in documentation. See [#847](https://github.com/redhat-developer/vscode-xml/pull/847).
+ * Update documentation with document link for `xi:include/@href`. See [#846](https://github.com/redhat-developer/vscode-xml/pull/846).
```

Comment:
I don't think these are worthwhile listing in the changelog, since most of these are documentation for features listed in the features section.